### ISSUE TITLE
Table sorting is not accessible

### ISSUE THREAD

**Issue by pfrenssen** [kevinpapst/kimai2#3646]
### Describe the problem

The sorting of datatables cannot be understood or controlled by accessibility software such as screen readers. This makes it impossible for users with visual / motor disabilities to change the sorting on the tables.

### Describe the solution you'd like

Implement the relevant accessibility attributes, ref. https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-sort
